sonaranalyzer-dotnet/rspec/cs/S4426_c#.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Cryptographic keys should not be too short",
+  "title": "Cryptographic keys should be robust",
```

Comment:
Since we are reverting changes related to this rule, I think we can also revert the title change here, as this change should be done when updating the implementation of the rule